### DISCUSSION TITLE
Rotating element with mouse movement

### DISCUSSION THREAD

**Discussion by anta199** [google/model-viewer#3160] [Q&A]
Hello,

Is there a trivial way to trigger the model rotation by mouse movement instead of grabbing?
I would like to track the mouse position on the browser to rotate the element.

Thanks in advance.

**Comment by samaneh-kazemi** [comment]
Hi @anta199, thanks for reaching out. You can try setting a mouse listener and whenever that changes, update camera orbit. Check our [panning example](https://modelviewer.dev/examples/stagingandcameras/#panning) which covers lots of listeners and camera changes.

**Comment by anta199** [comment]
Hello @samaneh-kazemi! Thanks a lot for taking time to help. This seems really useful.
I’m a total newbie on this so I was wondering if there’s a reference or example of how this you propose would be implemented? I don’t feel capable of doing this from scratch.

But thanks a lot again!

**Comment by samaneh-kazemi** [comment]
Hi @anta199, I recommend playing around with the example I shared in [glitch](https://glitch.com/ ). You can also look at other examples we have to get more familiar with our API. There are a lots of great free online JS courses. It  won't take long to get comfortable. You got this!
